modules/pricer/src/main/java/com/opengamma/strata/pricer/swap/DiscountingSwapLegPricer.java

```diff
@@ -593,8 +593,14 @@ CashFlows cashFlowEventsInternal(ResolvedSwapLeg leg, RatesProvider provider) {
   }
 
   //-------------------------------------------------------------------------
-  // explains the present value of the leg
-  void explainPresentValueInternal(ResolvedSwapLeg leg, RatesProvider provider, ExplainMapBuilder builder) {
+  /**
```

Comment:
Hmm, not sure if a method named `explainPresentValueInternal` should really be exposed as part of the public API. I'd suggest making it package-private again, then adding another method `explainPresentValue` which calls it passing in a clean builder. It could then return a fully built `ExplainMap`.